### DISCUSSION TITLE
Pull Request Approvals

### DISCUSSION THREAD

**Discussion by kmg-stripe** [Netflix/mantis#704] [General]
@Andyz26 @sundargates  - Hey all!  I think we are starting to hit a cadence of ~1 PR / week and will likely increase by a bit once we start working on some of the modernization tasks (e.g. Akka -> Pekka) we discussed back in June.  The last few PRs have been small bug fixes, mostly related to the DynamoDB and new leader election code.

I just wanted to check-in on how the Netflix folks feel regarding code reviews.  We don't want to keep bugging you for smaller bug fixes, but also want to make sure you all are in the loop.  Obviously, we'll want Netflix involved in the reviews for large or foundational changes, but are you all ok with a Stripe (@codyrioux , @sarahwada-stripe , @andresgalindo-stripe or I) approving smaller bug fixes (ex: https://github.com/Netflix/mantis/pull/703)?

**Comment by Andyz26** [comment]
I think it's okay for small bug fixes. The only thing I would ask for such changes is that you ping us if there are any potential compatibility-breaking changes (e.g. change on public class/interface). Sometimes, even if there is no usage of some public classes/methods, we still have usage internally, and I would like to avoid unexpected breaking changes there.

**Reply by crioux-stripe** [reply]
That makes sense and largely aligns with our thinking. We're trying to strike a balance between autonomy for our own changes, capitalizing on your attention, and being collaborative. 

> The only thing I would ask for such changes is that you ping us if there are any potential compatibility-breaking changes 
I think this is also a big priority for us as Netflix has a significantly more mature process for testing and evaluating these changes than we do.
